### ISSUE TITLE
manage.py error in SDK 1.9.17, old_dev_appserver removed

### ISSUE THREAD

**Issue by pv-g** [django-nonrel/djangoappengine#95]
With the new SDK 1.9.17, manage.py now raises an error : (on branch `master`)

```
Exception: Could not get appid. Is your app.yaml file missing? Error was: cannot import name old_dev_appserver
```

With branch `devappserver2`, it says :

```
File "[...]/djangoappengine/db/stubs.py", line 80, in setup_local_stubs
   from google.appengine.tools import dev_appserver_main
ImportError: cannot import name dev_appserver_main
```

**Comment by jacobg** [comment]
Yeah, I don't think the local stubs has been updated to work with the latest SDK version which removed the old devappserver. @aburgel Can you help us here please? Thanks!

**Comment by jacobg** [comment]
Regarding my last two comments, see the following new issue I created to address it. It includes patch fix:
https://github.com/django-nonrel/djangoappengine/issues/97

**Comment by interpegasus** [comment]
Hi everyone, After switching to devappserver2 the app(s) works fine again, but it is slower. It looks like models are validated on each request.
Before starting the server was very fast. And now it takes around 2-4 secs. 
Also according to logs Google development console, the app received more than 100000 requests in 24h period. Likely those are internal API request to the datastore from the devappserver2? 
Is there a way to optimize or solve this issue? Thanks

**Comment by Philoozushi** [comment]
Hi @interpegasus,
I run the same config as you, and I don't have such a problem.
It may be related with something more specific to your code.
Or do you have more details?

**Comment by Philoozushi** [comment]
Sorry @aburgel, I missed your last mention.
My `main.py` is still the same, with that little fix.
It was running well locally, and on AppEngine production.

But I just tried to remove it, and now it works well too, locally, and on production!
I'm unable to tell you why the error disappeared...

**Comment by interpegasus** [comment]
@Philoozushi Thanks for confirming that the same configuration works fine for you. 
The issue has been fixed on my end. And these were the steps I took:
* Installing latest version of all python libraries, Google App Launcher Python SDK. 
* Deleted and Git cloned again django nonrel and dependencies.
* Implementing an advanced cache strategy using GAE memcache.
The app is fast again, Datastore Read Operations seem to be optimal, and models are validated once instead of many times.

### This is a list of minor issues:
* manage.py shell works but database is empty and db operation are not persistent after exit()
* manage.py createsuperuser does not work on my end
* manage.py createsuperuser and manage.py changepassword do not work on my end. Unable to use the admin site without a superuser.

**Comment by dho825** [comment]
Hey guys, after updated appengine sdk to 1.9.18, and applying all the fixes and patches here, I can't seem to get syncdb to create my models to the local db (.gaedata/datastore)...

This is similar to the issue about createsuperuser not working by @npgauth 

I know that some of you guys were talking about some stubs.py/local db issues with the new dev_appserver and how it manages local connections, but I'm definitely not as well versed in the underlying SDK to even begin to debug this problem.

**Comment by gxela** [comment]
I get an error when running `python manage.py createsuperuser`

```
(RSA) ~/Projects/translator2048 $ python manage.py createsuperuser
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/alexgoretoy/Projects/translator2048/django/core/management/__init__.py", line 453, in execute_from_command_line
    utility.execute()
  File "/Users/alexgoretoy/Projects/translator2048/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/alexgoretoy/Projects/translator2048/django/core/management/__init__.py", line 263, in fetch_command
    app_name = get_commands()[subcommand]
  File "/Users/alexgoretoy/Projects/translator2048/django/core/management/__init__.py", line 109, in get_commands
    apps = settings.INSTALLED_APPS
  File "/Users/alexgoretoy/Projects/translator2048/django/conf/__init__.py", line 53, in __getattr__
    self._setup(name)
  File "/Users/alexgoretoy/Projects/translator2048/django/conf/__init__.py", line 48, in _setup
    self._wrapped = Settings(settings_module)
  File "/Users/alexgoretoy/Projects/translator2048/django/conf/__init__.py", line 132, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/Users/alexgoretoy/Projects/translator2048/django/utils/importlib.py", line 35, in import_module
    __import__(name)
  File "/Users/alexgoretoy/Projects/translator2048/translator2048/settings.py", line 10, in <module>
    from djangoappengine.settings_base import *
  File "/Users/alexgoretoy/Projects/translator2048/djangoappengine/settings_base.py", line 6, in <module>
    setup_env()
  File "/Users/alexgoretoy/Projects/translator2048/djangoappengine/boot.py", line 79, in setup_env
    setup_project()
  File "/Users/alexgoretoy/Projects/translator2048/djangoappengine/boot.py", line 140, in setup_project
    from .utils import have_appserver, on_production_server
  File "/Users/alexgoretoy/Projects/translator2048/djangoappengine/utils.py", line 25, in <module>
    "Error was: %s" % e)
Exception: Could not get appid. Is your app.yaml file missing? Error was: cannot import name old_dev_appserver
```
here is my app.yaml

```
application: translator2048
version: 1
runtime: python27
api_version: 1
threadsafe: yes

builtins:
- remote_api: on

inbound_services:
- warmup

handlers:
- url: /_ah/queue/deferred
  script: djangoappengine.deferred.handler.application
  login: admin

- url: /_ah/stats/.*
  script: djangoappengine.appstats.application

- url: /media/admin
  static_dir: django/contrib/admin/media
  expiration: '0'

- url: /.*
  script: djangoappengine.main.application
```

**Comment by dho825** [comment]
@gxela from the logs it seems you have not migrated over to using devappserver2.py since the new SDK, old_dev_appserver.py has been removed. I would recommend pulling from the devappserver2 branch and following the patches above to see if that fixes things

**Comment by gxela** [comment]
I just found this new way to run Django on GAE https://github.com/potatolondon/djangae
